app/javascript/CommentSubscription/CommentSubscription.jsx

```diff
@@ -107,16 +110,21 @@ export class CommentSubscription extends Component {
           <Button
             variant="outlined"
             onClick={(_event) => {
-              if (subscribed) {
-                onUnsubscribe(COMMENT_SUBSCRIPTION_TYPE.NOT_SUBSCRIBED);
-                this.setState({
-                  subscriptionType: COMMENT_SUBSCRIPTION_TYPE.ALL,
-                });
+              if (userStatus == "logged-in") {
```

Comment:
```suggestion
              if (userStatus === "logged-in") {
```
It may not matter too much in this instance, but we tend to favor the strict equality operator (`===`).

If we start using the `isLoggedIn` prop here instead, then we can get rid of the operator altogether:
```suggestion
              if (isLoggedIn) {
```